### DISCUSSION TITLE
Changing plot titles and font size possible?

### DISCUSSION THREAD

**Discussion by foxtrotduck** [gavinsimpson/gratia#204] [Q&A]
When using draw for a multi-plot gam, is it possible to change the name of the variables (e.g., from "air_press" to "Air Pressure") and/or change font size for the plots? See plot below created with

```
model <- gam(data = data, var ~ s(air_press, k = 4) + 
                      s(turb, k = 4)...
draw(model)
```

![000024](https://user-images.githubusercontent.com/122929380/212998317-94a6afcb-2618-49e7-8c74-446c36bed11f.png)

**Comment by gavinsimpson** [comment]
The font size is controlled through the theme, which is standard ggplot stuff. Just remember to use the `&` operator when you modify the theme so that is applies to all plots in the patchwork

```r
draw(m) & theme_grey(base_size = 9)
```

for example would set the base font size to 9 points, while still using the default theme.

As for the titles, axis labels, no, that's not something that can be changed. If you want full control, using

```r
smooth_estimates(m, smooth = "s(air_press)") |>
  draw(xlab = "Air Pressure", title = "s(Air Pressure)")
```

for more control and

```r
smooth_estimates(m, smooth = "s(air_press)") |>
  add_confint() |>
  ggplot(....)
```

for complete control. You can save those plots and then combine them into a patchwork using `wrap_plots()`.

I could look into allowing the user to pass a vector of titles for the plots, but that is not high on my priority list.
